src/test/java/com/google/api/codegen/testdata/csharp/csharp_library.baseline

```diff
@@ -2842,6 +2842,60 @@ namespace Google.Example.Library.V1.Tests
             mockGrpcClient.VerifyAll();
         }
 
+        [Fact]
+        public void CreateShelf2()
+        {
+            Mock<LibraryService.LibraryServiceClient> mockGrpcClient = new Mock<LibraryService.LibraryServiceClient>(MockBehavior.Strict);
+            mockGrpcClient.Setup(x => x.CreateLabelerClient())
+                .Returns(new Mock<Labeler.LabelerClient>().Object);
+            mockGrpcClient.Setup(x => x.CreateOperationsClient())
+                .Returns(new Mock<Operations.OperationsClient>().Object);
+            Shelf expectedResponse = new Shelf
+            {
+                ShelfName = new ShelfName("[SHELF_ID]"),
+                Theme = "theme110327241",
+                InternalTheme = "internalTheme792518087",
+            };
+            // TODO: Add verification of request object
+            mockGrpcClient.Setup(x => x.CreateShelf(It.IsAny<CreateShelfRequest>(), It.IsAny<CallOptions>()))
+                .Returns(expectedResponse);
+            LibraryServiceClient client = new LibraryServiceClientImpl(mockGrpcClient.Object, null);
+            CreateShelfRequest request = new CreateShelfRequest
+            {
+                Shelf = new Shelf(),
+            };
+            Shelf response = client.CreateShelf(request);
+            Assert.Equal(expectedResponse, response);
```

Comment:
Assert.Same in this case, as we don't expect any interference?

Reply:
Good idea, again, please can I make this change in the next PR?